### DISCUSSION TITLE
Update beam-community.org to use elixir instead of rails

### DISCUSSION THREAD

**Discussion by jsegal205** [beam-community/beam-community.org#25] [Ideas]
As a resource for elixir and erlang, [beam-community.org repo](https://github.com/beam-community/beam-community.org) should consider updating to use elixir instead of rails. There aren't any current persistence needs for the site, so we might not even need elixir and or rails and could achieve the same marketing site as a vanilla html page.

A non-db set up elixir phoenix app can be initialized with 
```sh
mix phx.new beam-community.org --no-context --no-ecto
```

[mix phx.new docs](https://hexdocs.pm/phoenix/Mix.Tasks.Phx.New.html)

Things to think about with this migration - 
- where is it hosted? 
   - will this work with jekyll?
   - should we update hosting during this migration?

**Comment by doomspork** [comment]
Just as a point of clarification: This website isn't built with Rails. It's a static site generated by Jekyll. I don't think firing up Phoenix and deploying it for a single HTML page is the right approach 🤔

**Reply by jsegal205** [reply]
TY for this clarification, I'm clearly not well versed in how Jekyll works.

**Comment by doomspork** [comment]
Closing this following the convo in Slack that the juice is probably not worth the squeeze. If we want to change this in the future we could probably get by with an Elixir passed static generator but probably a lower priority than effort on any of our other repos.
